src/with-props.js

```diff
@@ -178,5 +181,8 @@ export function getProps (elem) {
 }
 
 export function setProps (elem, props) {
+  if (typeof props === 'function') {
```

Comment:
we should have for these cases general facade-lang.js file

`typeof props === 'function'` => `isFunction(props)` etc